contracts/UFragmentsPolicy.sol

```diff
@@ -91,28 +91,29 @@ contract UFragmentsPolicy is Ownable {
         uint256 cpi;
         bool cpiValid;
         (cpi, cpiValid) = cpiOracle.getData();
-        require(cpiValid);
 
         uint256 targetRate = cpi.mul(10 ** DECIMALS).div(baseCpi);
 
         uint256 exchangeRate;
         bool rateValid;
         (exchangeRate, rateValid) = marketOracle.getData();
-        require(rateValid);
 
         if (exchangeRate > MAX_RATE) {
             exchangeRate = MAX_RATE;
         }
 
-        int256 supplyDelta = computeSupplyDelta(exchangeRate, targetRate);
+        int256 supplyDelta = 0;
```

Comment:
Since we include `exchangeRate` and `cpi` in the rebase log message, we should probably initialize them to 0 the same way we do for `supplyDelta` here.

Functionally it should be the same, but nice to not rely on default values.

Reply:
Per our offline discussion, If value is invalid the oracles return zero. It is useful to have the valid value in the log, if only one is invalid.